src/main/java/dev/slimevr/config/serializers/BooleanMapDeserializer.java

```diff
@@ -0,0 +1,4 @@
+package dev.slimevr.config.serializers;
+
+public class BooleanMapDeserializer extends MapDeserializer<Boolean> {
```

Comment:
Add class-level comment on this and the similar classes that says that it is here just to parameterize the MapDeserializer in the `@JsonDeserialize` annotation